packages/cadl-ranch-specs/http/resiliency/srv-driven/main.tsp

```diff
@@ -25,9 +25,12 @@ using TypeSpec.Http;
 })
 @service
 @server(
-  "http://localhost:3000/resiliency/service-driven/client:v2/service:{serviceDeploymentVersion}/api-version:{apiVersion}",
+  "{endpoint}/resiliency/service-driven/client:v2/service:{serviceDeploymentVersion}/api-version:{apiVersion}",
   "Testserver endpoint",
   {
+    @doc("Need to be set as 'http://localhost:3000' in client.")
+    endpoint: url,
```

Comment:
Should we set a default value for it ?

Reply:
I don't think we has client-default in tsp?